treeherder/model/migrations/0026_bugscache_add_dupe_of_and_processed_update.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.1.12 on 2021-09-21 13:51
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('model', '0025_remove_bugscache_os'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='bugscache',
+            name='dupe_of',
+            field=models.PositiveIntegerField(null=True),
+        ),
+        migrations.AddField(
+            model_name='bugscache',
+            name='processed_update',
+            field=models.BooleanField(default=True),
+        ),
+    ]
```

Comment:
These two migrations could probably be combined into one.

Reply:
We could combine all existing migrations into one (like it has been done in the past) to got from 26 migrations to 1 a few weeks after this has landed, or do this now either with all commits squashed or one commit at the end squashing the migrations. Because these the migrations are for different kind of changes, the first option looks preferable to me.

Reply:
Sure, that's fine.